Front/app/ns_modules/ns_form/NsFormsModuleGit.js

```diff
@@ -242,16 +242,27 @@ define([
 
         $(this.formRegion).find('textarea').on("keypress", function(e) {
             var maxlen = 250;
+            var self = this;
             if ($(this).val().length > maxlen) {
-              _this.showAlertforMaxLength();
+               _this.showErrorForMaxLength(this);
+               setTimeout(function(){ 
+                _this.cleantextAreaAfterError(self);
+
+              }, 3000);
               return false;
             }  
         });
         $(this.formRegion).find('textarea').on('keyup', function (e) {
               var maxlen = 250;
+              var strval = $(this).val();
+              var self = this;
               if ($(this).val().length > maxlen) {
-                _this.showAlertforMaxLength(); 
-                $(this).val('');
+                 _this.showErrorForMaxLength(this);
+                 var res = strval.substring(0, 250);
+                $(this).val(res);
+                setTimeout(function(){ 
+                _this.cleantextAreaAfterError(self);
```

Comment:
no timeout, you have to clean error message after re edition with text length < maxlength